backend/engine/plugins/trivy_sca/main.py

```diff
@@ -0,0 +1,63 @@
+"""
+trivy SCA plugin
+"""
+import json
+import subprocess
+from engine.plugins.lib import utils
+from engine.plugins.lib.trivy_common.generate_locks import check_package_files
+from engine.plugins.lib.trivy_common.parsing_util import convert_output
+from engine.plugins.lib.trivy_common.parsing_util import parse_output
+
+logger = utils.setup_logging("trivy_sca")
+
+NO_RESULTS_TEXT = "no supported file was detected"
+
+
+def execute_trivy_lock_scan(path: str, include_dev: bool):
+    logger.info(f"Scanning lock-files. Dev-dependencies: {include_dev}")
+    args = ["trivy", "fs", "--include-dev-deps", path, "--format", "json"]
+    if not include_dev:
```

Comment:
@davakos this code was a proposal of mine in a [previous comment](https://github.com/WarnerMedia/artemis/pull/80#discussion_r1465815550) because I wasn't sure if arg order mattered. @faizan12123 does the command work if `--include-dev-deps` is the last arg or is the order important?

Reply:
good call, if order matters please leave as-is

Reply:
going to test that out, it should not matter but let me double check